app.js

```diff
@@ -24,13 +28,52 @@ const config = {
     api_user: 'admin',
     api_pass: 'admin',
     play: false,
-    publish: false,
-    secret: 'nodemedia2017privatekey'
+    publish: process.env.SECURE_PUBLISH, // enables sign parameter to be used for server
+    secret: 'radiantNodeMediaServer2019'
+  },
```

Comment:
is this secret something we want in the .env?

Reply:
yes I will change it and put it in env